### ISSUE TITLE
Segment event reporting not working for cloud

### ISSUE THREAD

**Issue by cgardens** [airbytehq/airbyte#5482]
## Current Behavior
* We expect the cloud version of the app to push events to segment. It seems like no events are being sent at all. The env variables that should govern this are set in the `.env` [file](https://github.com/airbytehq/airbyte-cloud/blob/master/.env#L55-L56) in `airbyte-cloud`. Not sure these just aren't getting propagated to the runtime environment or what.

**Comment by davinchia** [comment]
Looks like this wasn't set properly:

```
(⎈ |gke_prod-ab-cloud-proj_us-west3_prod-gke-cluster:default)
➜  airbyte git:(master) k exec -it airbyte-cloud-server-67c6c67db5-74kz8 -n ab -c cloud-server env | grep tracking
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead.
```

**Comment by davinchia** [comment]
papercups, fullstory and openreplay were also not turned on - I've enabled them.

@cgardens I'm not sure how to properly QA that the right events are coming through. Any thoughts?

**Comment by cgardens** [comment]
yes! there is a [debugger](https://app.segment.com/airbyte/sources/web_app/debugger) in segment that makes this easy. You might need to mess around a bit to filter out events from OSS.

**Comment by davinchia** [comment]
This is deployed. It looks like we are only getting a subset of the calls instead of the full segment suite.

**Comment by davinchia** [comment]
This is fixed as of #5597. Change were into dev yesterday and prod today.

<img width="1823" alt="Screen Shot 2021-09-07 at 12 40 22 PM" src="https://user-images.githubusercontent.com/9772859/132285619-24fede1d-eed6-435c-b657-9b6d76bc3a21.png">
